clevertap-core/src/test/java/com/clevertap/android/sdk/utils/CTCachesTest.kt

```diff
@@ -2,12 +2,14 @@ package com.clevertap.android.sdk.utils
 
 import com.clevertap.android.sdk.TestLogger
 import org.junit.After
+import org.junit.Ignore
 import org.junit.Test
 import kotlin.test.assertEquals
 
+@Ignore
```

Comment:
Why Ignore

Reply:
Fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/d5c573fd09510eedd1d9321a561611e15ac7a569